xapian-applications/omega/docs/omegascript.rst

```diff
@@ -542,6 +542,15 @@ $set{OPT,VALUE}
 	Similarly, ``XFOO:stemmer`` specifies the stemmer to use for field
 	``XFOO``, with ``stemmer`` providing a default.
 
+$seterror{error_msg}
```

Comment:
The arguments in this document are conventional upper case (i.e. `ERROR_MSG` or perhaps better to avoid the abbreviation and say `ERROR_MESSAGE` or `MESSAGE`).